pennylane/qinfo/transforms.py

```diff
@@ -83,17 +82,24 @@ def vn_entropy(qnode, wires, base=None):
 
     **Example**
 
-        .. code-block:: python
+    It is possible to obtain the entropy of a subsystem from a :class:`.QNode` returning a :func:`~.state`.
 
-            dev = qml.device("default.qubit", wires=2)
-            @qml.qnode(dev)
-            def circuit(x):
-                qml.IsingXX(x, wires=[0, 1])
-                return qml.state()
+    .. code-block:: python
+
+        dev = qml.device("default.qubit", wires=2)
+        @qml.qnode(dev)
+        def circuit(x):
+            qml.IsingXX(x, wires=[0, 1])
+            return qml.state()
 
     >>> vn_entropy(circuit, wires=[0])(np.pi/2)
     0.6931472
 
+    The function is differentiable with backpropagation for all interfaces, e.g.:
+
+    >>> param = pennylane.numpy.array(np.pi/4, requires_grad=True)
```

Comment:
I think it's safe to assume that `pennylane.numpy` is already imported.
```suggestion
    >>> param = np.array(np.pi/4, requires_grad=True)
```